Sources/App/Settings/DebugSettingsViewController.swift

```diff
@@ -374,13 +375,29 @@ class DebugSettingsViewController: HAFormViewController {
             )
 
             alert.addAction(UIAlertAction(title: L10n.okLabel, style: .default, handler: { _ in
-//                SentrySDK.crash()
+                //                SentrySDK.crash()
             }))
 
             self?.present(alert, animated: true, completion: nil)
             alert.popoverPresentationController?.sourceView = cell.formViewController()?.view
         }
 
+        if #available(iOS 16.4, *) {
+            section <<< ButtonRow {
+                $0.title = L10n.Settings.Developer.MockThreadCredentialsSharing.title
+            }.onCellSelection { [weak self] _, _ in
+                let viewController = UIHostingController(
+                    rootView: ThreadCredentialsSharingView(
+                        viewModel: .init(
+                            server: ServerFixture.standard,
+                            threadClient: MockThreadClientService()
```

Comment:
maybe let's call it 'Simulator…' rather than 'Mock…' since it is not really mocking anything